cfme/tests/generic_objects/test_instances.py

```diff
@@ -284,3 +342,41 @@ def cleanup():
     ).success
 
     assert gen_obj_def_import_export.exists
+
+
+@pytest.mark.meta(automates=[1729341, 1743266])
+def test_generic_object_with_service_button(
+        appliance, generic_object, textbox_dialog, button_with_dialog
+):
+    """
+    Bugzilla:
+        1729341
+        1743266
+
+    Polarion:
+        assignee: jdupuy
+        initialEstimate: 1/6h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: GenericObjects
+    """
+    textbox_dialog, TextBoxDialogView = textbox_dialog
+    # add generic object to service
+    myservice = MyService(appliance, name=generic_object.associations.get("services")[0].name)
+    myservice.add_generic_object(generic_object)
+    # add myservice as part of the generic_objecct
+    generic_object.my_service = myservice
```

Comment:
If we are using `generic_object` global fixture then... this is done by that only

https://github.com/ManageIQ/integration_tests/blob/d776a12dd12e16475ac55d369a27ebecd29a6e7e/cfme/fixtures/generic_object.py#L39